cookbooks/cdo-apps/templates/default/unicorn.sh.erb

```diff
@@ -34,7 +36,7 @@ run () {
   if [ "$(id -un)" = "$AS_USER" ]; then
     eval $1
   else
-    su -c "$1" - $AS_USER
+    su -c "$1" $AS_USER
```

Comment:
Turns out we need to remove the `-` option in order to start Unicorn service from `root` - otherwise, the Bundler settings (exported above) are not passed along to the `su` process, and the `bundle`-environment command can fail.